build-tools/debian-metadata/xamarin.android.install

```diff
@@ -0,0 +1,2 @@
+bin/Debug/bin/xabuild		/usr/bin
```

Comment:
This... is an issue with the full builds on Jenkins, which I didn't consider from my test builds on my laptop.

I guess it's a policy decision - do we want debug symbols or not? Could we provide both, meaningfully?